### ISSUE TITLE
Kernel restarting error.

### ISSUE THREAD

**Issue by shcrela** [spyder-ide/spyder#15202]
### What steps will reproduce the problem?

Pressing Ctrl+. while on the console window.

Opening new console could be a workaround.

### Traceback
```python-traceback
Traceback (most recent call last):
  File "/home/dejan/anaconda3/lib/python3.9/site-packages/spyder/widgets/browser.py", line 332, in contextMenuEvent
    self.context_menu.popup(event.globalPos())
AttributeError: 'NoneType' object has no attribute 'popup'
```

## Versions

* Spyder version: 5.0.0 
* Python version: 3.9.0
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Linux 4.15.0-140-generic

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.22.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.0.1 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.8.1 (OK)
pylint >=1.0                  :  2.7.4 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
rtree >=0.8.3                 :  0.9.7 (OK)
setuptools >=39.0.0           :  52.0.0.post20210125 (OK)
sphinx >=0.6.6                :  3.5.3 (OK)
spyder_kernels >=2.0.1,<2.1.0 :  2.0.1 (NOK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
xdg >=0.26                    :  0.27 (OK)
zmq >=17                      :  22.0.3 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=2.0.0            :  3.4.1 (OK)
numpy >=1.7                   :  1.20.2 (OK)
pandas >=1.1.1                :  1.2.3 (OK)
scipy >=0.17.0                :  1.6.2 (OK)
sympy >=0.7.3                 :  None (NOK)
```

**Comment by nranthony** [comment]
I get a message on startup that `spyder_kernels` is missing:
![image](https://user-images.githubusercontent.com/16306836/114185706-0491ed00-9914-11eb-89b5-ec85eeca7acb.png)

I also see that you have `spyder_kernels >=2.0.1,<2.1.0 :  2.0.1 (NOK)`

I think there's a typo, shouldn't it be `spyder-kernels` ?

**Comment by shcrela** [comment]
> I get a message on startup that `spyder_kernels` is missing:
> ![image](https://user-images.githubusercontent.com/16306836/114185706-0491ed00-9914-11eb-89b5-ec85eeca7acb.png)
> 
> I also see that you have `spyder_kernels >=2.0.1,<2.1.0 : 2.0.1 (NOK)`
> 
> I think there's a typo, shouldn't it be `spyder-kernels` ?

I dont think your issue is related, but I did have that one too. Here is a response to your issue on stackoverflow:
https://stackoverflow.com/questions/66952832/spyder-5-missing-dependencies-spyder-kernels-version-error/66959690#66959690
And yes, I believe the underscore is a typo (it should be a dash)

**Comment by ccordoba12** [comment]
Hey @shcrela, thanks for reporting. Your problem is a duplicate of issue #15098 and it'll be fixed in our next version (5.0.1).

@nranthony, as @shcrela mentioned, yours is a duplicate of issue #15091 and it'll be fixed in 5.0.1 as well.
